ui/app/actions.js

```diff
@@ -1806,6 +1832,12 @@ function hideAlert () {
   }
 }
 
+function qrCodeDetected (qrCodeData) {
```

Comment:
This is a pretty small, nit-picky request, but could you make it explicit what properties are in qrCodeData? In the absence of both tests and formal documentation for this file, make properties explicit makes the code a bit more self documenting. See, for example, the `showQrView`, `updateSendTo` or `setCurrentCurrency` functions in this file.

In this case I am thinking of some variation of:
```
function qrCodeDetected ({ type, values }) {
  return {
    type: actions.QR_CODE_DETECTED,
    value: { type, values },
  }
```

Reply:
Another option might be to add a docblock to the function

Reply:
Ended up following @whymarrh's suggestion. Thanks!